scrapy/extensions/feedexport.py

```diff
@@ -131,18 +132,28 @@ def __init__(self, uri, access_key=None, secret_key=None):
     @classmethod
     def from_crawler(cls, crawler, uri):
         return cls(uri, crawler.settings['AWS_ACCESS_KEY_ID'],
-                   crawler.settings['AWS_SECRET_ACCESS_KEY'])
+                   crawler.settings['AWS_SECRET_ACCESS_KEY'],
+                   crawler.settings.get('FEED_STORAGE_S3_ACL'))
 
     def _store_in_thread(self, file):
         file.seek(0)
         if self.is_botocore:
+            kwargs = dict()
+            if self.acl:
+                kwargs.update(dict(ACL=self.acl))
+
             self.s3_client.put_object(
-                Bucket=self.bucketname, Key=self.keyname, Body=file)
+                Bucket=self.bucketname, Key=self.keyname, Body=file,
+                **kwargs)
```

Comment:
Any reasons to use `**kwargs` instead of simple method call with a named argument?

Reply:
I'm just trying to avoid changing the previous behavior. I'm not sure about the side-effects of explicitly passing `ACL` or `policy` arguments even if they're defined as `None`. That's because the `ACL` argument, for example, is not a named argument, but loaded via `kwargs` in the `put_object` method.

Reply:
Ok. Got it.
@dangra can you share some thought here as well?